nautobot/extras/models/models.py

```diff
@@ -944,6 +953,187 @@ def log(
+    )
+    interval = models.CharField(choices=JobExecutionType, max_length=255)
+    args = models.JSONField(blank=True, default=list, encoder=NautobotKombuJSONEncoder)
+    kwargs = models.JSONField(blank=True, default=dict, encoder=NautobotKombuJSONEncoder)
+    queue = models.CharField(
+        max_length=200,
+        blank=True,
+        null=True,
+        default=None,
+        verbose_name="Queue Override",
+        help_text="Queue defined in CELERY_TASK_QUEUES. Leave None for default queuing.",
+    )
+    one_off = models.BooleanField(
+        default=False,
+        verbose_name="One-off Task",
+        help_text="If True, the schedule will only run the task a single time",
+    )
+    start_time = models.DateTimeField(
+        verbose_name="Start Datetime",
+        help_text="Datetime when the schedule should begin triggering the task to run",
+    )
+    enabled = models.BooleanField(
+        default=True,
+        verbose_name="Enabled",
+        help_text="Set to False to disable the schedule",
+    )
+    last_run_at = models.DateTimeField(
+        auto_now=False,
+        auto_now_add=False,
+        editable=False,
+        blank=True,
+        null=True,
+        verbose_name="Last Run Datetime",
+        help_text="Datetime that the schedule last triggered the task to run. "
+        "Reset to None if enabled is set to False.",
+    )
+    total_run_count = models.PositiveIntegerField(
+        default=0,
+        editable=False,
+        verbose_name="Total Run Count",
+        help_text="Running count of how many times the schedule has triggered the task",
+    )
+    date_changed = models.DateTimeField(
+        auto_now=True,
+        verbose_name="Last Modified",
+        help_text="Datetime that this PeriodicTask was last modified",
+    )
+    description = models.TextField(
+        blank=True,
+        verbose_name="Description",
+        help_text="Detailed description about the details of this Periodic Task",
+    )
+    user = models.ForeignKey(
+        to=settings.AUTH_USER_MODEL,
+        on_delete=models.SET_NULL,
+        related_name="+",
+        blank=True,
+        null=True,
+        help_text="User that requested the schedule",
+    )
+    approved_by_user = models.ForeignKey(
+        to=settings.AUTH_USER_MODEL,
+        on_delete=models.SET_NULL,
+        related_name="+",
+        blank=True,
+        null=True,
+        help_text="User that approved the schedule",
+    )
+    approval_required = models.BooleanField(default=False)
+    approved_at = models.DateTimeField(
+        auto_now=False,
+        auto_now_add=False,
+        editable=False,
+        blank=True,
+        null=True,
+        verbose_name="Approval date/time",
+        help_text="Datetime that the schedule was approved",
+    )
+
+    objects = ScheduledJobExtendedQuerySet.as_manager()
+    no_changes = False
+
+    def __str__(self):
+        return "{0.name}: {0.interval}".format(self)
```

Comment:
A bit of a non-standard style here - perhaps?

```suggestion
        return f"{self.name}: {self.interval}"
```